### DISCUSSION TITLE
Preview7 build freeze on Windows

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#11243] [Q&A]
Building on Linux works fine. When I build for windows on Windows VM, sometimes the build freezes.

When I try to cancel, it stays stuck at

    war ning MSB4220: Waiting for the currently executing task "AvaloniaStatsTask" to cancel.

Canceled the task and that message keeps displaying non-stop until I kill all dotnet processes.

I re-run the same build and it works, so it seems pretty random.

Had no problem like this before, so it might be a regression in Preview7?

**Comment by workgroupengineering** [comment]
To avoid this issue Set Envirorment variabile AVALONIA_TELEMETRY_OPTOUT to 1. see #11116 for dettails.
